packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -130,7 +130,25 @@ export const messages = {
   force_stop_apex_debug_logging:
     'SFDX: Turn Off Apex Debug Log for Replay Debugger',
   isv_debug_bootstrap_step1_create_project:
-    'SFDX: ISV Debugger Setup, Step 1 of 2: Create Project',
-  isv_debug_bootstrap_step1_configure_project:
-    'SFDX: ISV Debugger Setup, Step 2 of 2: Configure Project'
+    'SFDX: ISV Debugger Setup, Step 1 of 7: Creating project',
+  isv_debug_bootstrap_step2_configure_project:
+    'SFDX: ISV Debugger Setup, Step 2 of 7: Configuring project',
+  isv_debug_bootstrap_step3_retrieve_org_source:
+    'SFDX: ISV Debugger Setup, Step 3 of 7: Retrieving unpackaged Apex code',
+  isv_debug_bootstrap_step4_convert_org_source:
+    'SFDX: ISV Debugger Setup, Step 4 of 7: Converting unpackaged Apex code',
+  isv_debug_bootstrap_step5_list_installed_packages:
+    'SFDX: ISV Debugger Setup, Step 5 of 7: Querying for installed packages',
+  isv_debug_bootstrap_step6_retrieve_packages_source:
+    'SFDX: ISV Debugger Setup, Step 6 of 7: Retrieving packages',
+  isv_debug_bootstrap_step7_convert_package_source:
+    'SFDX: ISV Debugger Setup, Step 7 of 7: Converting package: %s',
+  isv_debug_bootstrap_processing_package: 'Processing package: %s',
+  isv_debug_bootstrap_open_project:
+    'Opening Project in new Visual Studio Code Window',
```

Comment:
@ruthemmanuelle the last two options only appear in the output.

Reply:
Let's use sentence-case capitalization in the last one, for consistency:

'Opening project in new Visual Studio Code window'